index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Subrat Dash!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">    
+</head>
+<body>
+    <header>
+        <h2>Welcome</h2>
+        <i class="fa fa-bars fa-2x"></i>
+    </header>
+    <main>
+        <h1>Hey there. I'm <br/>Subrat <br/><span class="profession">I'm a software<br/>developer</span></h1>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </main>
+    <footer>
+        <div class="socialIcons">
+            <i class="fa fa-twitter fa-2x"></i>
+            <i class="fa fa-linkedin fa-2x"></i>
+            <i class="fa fa-github fa-2x"></i>
+        </div>
+        <i class="fa fa-chevron-down "></i>
+    </footer>

```

Comment:
- [ ] Hi your layed out design is looking great 👏. Please make sure that have all the headline content wrapped inside a `section`  tag.
- [ ] you don't need a `footer` tag here for the icons. the `footer` tag usually stores content related to the bottom of the page. so in this case it is not semantically correct. I hope you understand.

Something like this would be much better.
```suggestion
    <section>
      <h1>Hey there. I'm <br />Subrat <br /><span class="profession">I'm a software<br />developer</span></h1>
      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like
        what you see and have a project you need coded,
        don’t hestiate to contact me.</p>
      <div class="socialIcons">
        <i class="fa fa-twitter fa-2x"></i>
        <i class="fa fa-linkedin fa-2x"></i>
        <i class="fa fa-github fa-2x"></i>
      </div>
      <i class="fa fa-chevron-down "></i>
    </section>

```